WaterLog_Backend/Procedures.cs

```diff
@@ -47,14 +45,14 @@ public async Task triggerInsert(ReadingsEntry value)
                 //Updateleakagestatus
                 if (await _db.SegmentLeaks.AnyAsync(leak => leak.SegmentsId == segmentid))
                 {
-                    SegmentLeaksEntry latestEntry = await _db.SegmentLeaks.Where(leak => leak.SegmentsId == segmentid).OrderByDescending(lk => lk.LatestTimeStamp).FirstAsync();
+                    SegmentLeaksEntry latestEntry = await _db.SegmentLeaks.Where(leak => leak.SegmentsId == segmentid && leak.ResolvedStatus == "unresolved").OrderByDescending(lk => lk.LatestTimeStamp).FirstAsync();
```

Comment:
Long line I think can be entered after each statement e.g. .OrderByDescending.

Reply:
As above. I have seperated longer lines into smaller lines under each other.